Formula/s/scalr.rb

```diff
@@ -0,0 +1,21 @@
+class Scalr < Formula
+  desc "\"scalr\" is a command-line tool that communicates directly with the Scalr API"
+  homepage "https://github.com/Scalr/scalr-cli"
+  url "https://github.com/Scalr/scalr-cli/archive/refs/tags/v0.15.5.tar.gz"
+  sha256 "23a554dc856fe1718e92cfa4108b856ddcb9facd5197259026e45c2037dc978f"
+  license "Apache-2.0"
+
+  depends_on "gnu-sed" => :build
+  depends_on "go" => :build
+
+  def install
+    ENV["VERSION"] = version
```

Comment:
By the way, I did this (setting ENV var) intentionally because the official build does the same. Tried to emulate the build as much as possible.